### PR TITLE
AGhost Protection

### PR DESCRIPTION
**Pull request by NGGJamie**
Staff can get robbed of all their items when they aghost to deal with an issue for too long. This is a quick fix that makes their character invisible and passable so that they won't get robbed and can safely leave to deal with issues.

### CODE REVIEW

File: code/modules/admin/admin_verbs.dm [hunk 1]
```diff
@@ -378,7 +378,14 @@ GLOBAL_PROTECT(admin_verbs_hideable)
...
 			log_admin("[key_name(usr)] re-entered corpse")
 			message_admins("[key_name_admin(usr)] re-entered corpse")
+		var/mob/living/M = ghost.mind.current
+		var/datum/status_effect/incapacitating/sleeping/S = M.IsSleeping()
+		if(S && !M.IsKnockdown() && !M.IsStun() && !M.IsParalyzed()) // Wake them up unless they're asleep for another reason
+			M.remove_status_effect(S)
+			M.set_resting(FALSE, TRUE)
 		ghost.can_reenter_corpse = 1 //force re-entering even when otherwise not possible
+		ghost.mind.current.invisibility = 0
+		ghost.mind.current.density = 1
```
Comment (Contrabang) [inline code/modules/admin/admin_verbs.dm:388]: Do we want initial for this density? Currently spirits are not dense, and this could make them dense

Comment (NGGJamie) [reply code/modules/admin/admin_verbs.dm:388]: I suppose I could add a check if they're a spirit, hold please.

**Review by Contrabang — APPROVED** [review]
LGTM

**Comment by emoats18** [comment]
Jamie is the supreme ruler of this and can merge whatever he pleases. Tally ho! And sucketh on mine peanuts peasants! HUZZAH!

**Pull request merged**